roles/pulp_common/defaults/main.yml

```diff
@@ -7,6 +7,8 @@ pulp_config_dir: '/etc/pulp'
 # foreseeable need for them to change the filename, only pulp_config_dir.
 pulp_settings_file: '{{ pulp_config_dir }}/settings.py'
 pulp_settings_file_local: '{{ pulp_config_dir }}/settings.local.py'
+pulp_aiohttp_fixtures_origin: '127.0.0.1'
+pulp_remote_fixtures_origin: 'https://fixtures.pulpproject.org/'
```

Comment:
If they're openly needed in pulp_devel, put the variables in the pulp_devel role.